src/main/js/PipelineEditorLink.jsx

```diff
@@ -46,6 +39,21 @@ class PipelineEditorLink extends React.Component {
             </Link>
         );
     }
+
+    @action
```

Comment:
but this doesn't actually have any `@observable` properties, so there should not be any need for `action` here at all